label_studio/khan/rbac/migrations/0001_initial.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.2.19 on 2023-07-26 18:29
```

Comment:
pardon my ignorance - what does this do and why should it be in version control?

Reply:
this is a db migration, so it is responsible for creating the database tables/columns/etc. it is generated by `manage.py makemigrations` and applied using `manage.py migrate` the latter of which is done automagically when label-studio starts. (this is actually a label-studio-ism, not a django-ism, and I'm personally on the fence about it, but nobody asked me 😉 ). These are basically python DDL.